projects/scion/microfrontend-platform/src/lib/host/manifest-registry/ɵmanifest-registry.ts

```diff
@@ -299,12 +299,12 @@ function coerceCapabilityParamDefinitions(capability: Capability, appSymbolicNam
   capability.requiredParams?.forEach(name => { // eslint-disable-line deprecation/deprecation
     params.push({name, required: true});
     const migration = `{ params: [{name: '${name}', required: true}] }`;
-    Beans.get(Logger).warn(`[DEPRECATION WARNING] The '${appSymbolicName}' application uses a deprecated API for declaring required parameters of a capability. The API will be removed in a future release. To migrate, declare parameters by using the 'Capability#params' property, as follows: ${migration}`, capability);
+    Beans.get(Logger).warn(`[DEPRECATION] The '${appSymbolicName}' application uses a deprecated API for declaring required parameters of a capability. The API will be removed in a future release. To migrate, declare parameters by using the 'Capability#params' property, as follows: ${migration}`, new LoggingContext(appSymbolicName), capability);
   });
   capability.optionalParams?.forEach(name => { // eslint-disable-line deprecation/deprecation
     params.push({name, required: false});
     const migration = `{ params: [{name: '${name}', required: false}] }`;
-    Beans.get(Logger).warn(`[DEPRECATION WARNING] The '${appSymbolicName}' application uses a deprecated API for declaring optional parameters of a capability. The API will be removed in a future release. To migrate, declare parameters by using the 'Capability#params' property, as follows: ${migration}`, capability);
+    Beans.get(Logger).warn(`[DEPRECATION] The '${appSymbolicName}' application uses a deprecated API for declaring optional parameters of a capability. The API will be removed in a future release. To migrate, declare parameters by using the 'Capability#params' property, as follows: ${migration}`, new LoggingContext(appSymbolicName), capability);
```

Comment:
If no symbolic name is specified via the logging context, the logger prefixes the log message with the symbolic name of the current application. However, the above log message refers to the application using a deprecated API, not the host application. Otherwise, the log message would be confusing as prefixed with the host's symbolic name, indicating that the host is using a deprecated API.

Reply:
Same here